docs/developer/getting-started.md

```diff
@@ -8,6 +8,7 @@ Make sure the following software is installed and added to the $PATH variable:
 
 * Docker 1.10+ ([installation manual](https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/))
```

Comment:
I think we should change this to Docker 1.13.1 since that's the minimum docker version in K8s right now.

Reply:
done